pdm/builders/base.py

```diff
@@ -90,7 +90,7 @@ def log_subprocessor(
         outstream.stop()
 
 
-class _Prefix:
+class _Prefix:  # noqa: SIM119
```

Comment:
`flake8-simplify` suggests using a dataclass here. that's not a terrible idea, but it's a larger refactor, so i've just suppressed it for now